hbcbot/commands.py

```diff
@@ -11,3 +11,28 @@ def calc_abv(args):
     except Exception:
         # not passed numbers
         return usage
+
+def brix_sg(args):
+    usage = 'Usage: .brix <Original BRIX> (<Final BRIX>)'
+    if len(args) < 1 and len(args) > 2:
+        return usage
+    if len(args) == 1:
+        brix = float(args[0])
+        try:
+            sg = (brix/(258.6-((brix/258.2)*227.1)))+1
+            return 'OG is %.3f' % sg
+        except Exception:
+            # not passed numbers
+            return usage
+    if len(args) == 2:
+        obrix = float(args[0])
+        fbrix = float(args[1])
+        try:
+            sg = (obrix/(258.6-((obrix/258.2)*227.1)))+1
+            fg = 1-(0.004493 * obrix )+(0.011774 * fbrix) + (0.00027581*obrix*obrix)-(0.0012717*fbrix*fbrix) - (0.00000728*obrix*obrix*obrix)+(0.000063293*fbrix*fbrix*fbrix)
```

Comment:
In which I realize I never put a linter in here. >:(

(I can fix it up later, don't worry about it)